scripts/switching_parallel.py

```diff
@@ -88,8 +112,12 @@
     # define lambda space
     lambs = np.linspace(1, 0, switching_length)
     # perform NEQ from QML to MM
-    ws_from_qml_to_mm = perform_switching(
-        sim, lambdas=lambs, samples=qml_samples, nr_of_switches=nr_of_switches
-    )
+    ws_from_qml_to_mm = []
```

Comment:
here the same as above.
If I am not mistaken I think we should revert these lines back to the previous state.